shared/common-adapters/icon.js.flow

```diff
@@ -3,7 +3,10 @@
+  type: 'GPG-export' | 'computer-big' | 'documentation' | 'keybase-update-pause' | 'keybase-update' | 'logo-15' | 'logo-128' | 'logo-256' | 'logo-32' | 'logo-64' | 'no-gpg' | 'paper-key' | 'phone-big' | 'phone-oblique' | 'phone-q-r-code-small' | 'phone-q-r-code' | 'phone-text-code-small' | 'phone-text-code' | 'placeholder-avatar' | 'terminal' | 'user-card'
+    | 'fa-custom-copy' | 'fa-custom-eye' | 'fa-custom-key'
+    | 'fa-icon-proof-good-followed' | 'fa-icon-proof-good-new' | 'fa-icon-proof-broken'
```

Comment:
can we prefix all these svgs with 'custom'. and update the docs? i just want to avoid any cross contamination w/ font awesome since we're making a merged font